documentation/library_guide/macros.rst

```diff
@@ -87,12 +87,12 @@ Macro                              Description
                                    If all parallel backends are disabled by setting respective macros to 0, algorithms
                                    with parallel policies are executed sequentially by the calling thread.
 ---------------------------------- ------------------------------
-``ONEDPL_USE_DPCPP_BACKEND``       This macro enables the use of the SYCL* policies.
+``ONEDPL_USE_DPCPP_BACKEND``       This macro enables the use of the device policies.
                                    When the macro is not defined (by default)
-                                   or evaluates to non-zero, SYCL policies are enabled.
+                                   or evaluates to non-zero, device policies are enabled.
                                    When the macro is set to 0 there is no dependency on
                                    the |dpcpp_cpp| and runtime libraries.
-                                   Trying to use SYCL policies will lead to compilation errors.
+                                   Trying to use device policies will lead to compilation errors.
 ---------------------------------- ------------------------------
 ``ONEDPL_USE_PREDEFINED_POLICIES`` This macro enables the use of predefined policies objects,
```

Comment:
Let's change to "... the use of the predefined device policy objects, such as ``dpcpp_default`` and ``dpcpp_fpga``." This clarification is important, because other predefined policies (``par_unseq`` etc.) are not affected by the macro.